docs/configuration/index.md

```diff
@@ -483,6 +484,14 @@ Instead use `recipientBaseUrlPattern` described in the table below.
 
 To use graphite as emitter set `druid.emitter=graphite`. For configuration details, see [Graphite emitter](../development/extensions-contrib/graphite.md) for the Graphite emitter Druid extension.
 
+##### Switching Emitter
+
+To use switching as emitter set `druid.emitter=switching`. 
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`druid.emitter.switching.emitters`|Map of feed to list of emitter modules to load that will be used for each feed, e.g., {"metrics":["http"], "alerts":["logging"]}|{}|
+|`druid.emitter.switching.defaultEmitters`|List of emitter modules to load that will be used if there is no emitter specifically designated to the event feed, e.g., ["logging","http"].|[]|
```

Comment:
so that event feed will go to all the default emitters?

Reply:
So this where naming gets weird. Default is probably the wrong name. since it could be read that no matter what every event goes to the default emitters but i see it as it as the list of emitters to use if there isn't a match for an event's feed

Example of how it's working as coded:
So with a hypothetical config and hypothetical event feed names, and emitters
```
druid.emitter.switching.defaultEmitters=[http, foo]
druid.emitter.switching.emitters={"metrics":["statsd", "logging"], "alerts":["bar"]}
```

if an event with feed `metrics` is emitted. it will only be emitted to `statsd` and `logging` emitters and will not go to the default emitters. 
if an event with feed `requestLog` it would be emitted to the `http` and `foo` emitters
if an event with feed `alerts` it would be emitted to `bar` emitter

Reply:
@abhishekagarwal87 One thing that can be done here is i add this per feed functionality onto the composing emitter instead of making a new one...

Reply:
I think the current structure is fine since the final JSON would get a bit unwieldy.  what do you think?

Reply:
ok, no complaints from me then